### DISCUSSION TITLE
Ability to duplicate Pods template would be great

### DISCUSSION THREAD

**Discussion by pmsteil** [pods-framework/pods#6612] [Q&A]
Want to be able to easily take an existing template and duplicate it to easily make changes to the template.

**Comment by Shelob9** [comment]
@pmsteil I just tested and can confirm that the [Duplicate Post](https://wordpress.org/plugins/duplicate-post/) plugin can be used to duplicated Pods Templates. Templates are posts in a private post type _pods_template.

Does this work for you?

**Comment by pglewis** [comment]
Reassigning to "Future Release".  

Is this potentially an enhancement?  Yes, another plugin can solve it, but it might be nice to have a button right there.

**Comment by pmsteil** [comment]
Sorry Josh, missed your comment... 

I am glad there is a workaround for this and I will probably use that plugin for other things anyway... :)

But I do think this should be there in the pods code base as well as I think you can currently duplicate a pod, so it would be consistent to be able to duplicate a pod template and probably even more useful for a template... 

thanks!

**Comment by Shelob9** [comment]
@pmsteil I agree it would be nice, but for now, using that plugin is one less thing for us to make, test, not break...

**Comment by pmsteil** [comment]
That's cool, as long as it is on the list for future, sounds good... :)

*--Patrick Steil |  *infraNET Complete Website Optimization
http://www.infranet.com/

Mobile: 940-391-9250
LinkedIn:  http://linkedin.com/in/pmsteil
